src/simulator/views/gui/simulator_config.py

```diff
@@ -395,23 +397,32 @@ def __init__(self, services: Services, mouse1_press_callbacks: List[Callable[[],
             frameColor=TRANSPARENT)
 
         # setup state & use saved state if possible
+        self.__state = None
         for so in self.__services.state.objects:
             if isinstance(so, SimulatorConfigState):
                 self.__state = so
                 cmd = self.__maps_option['command']
-                self.__maps_option.set(self.__map_keys.index(so.mp))
-                self.__maps_option['command'] = cmd
-                self.__algorithms_option.set(self.__algorithm_keys.index(so.algo))
-                self.__animations_option.set(self.__animation_keys.index(so.ani))
-                self.__update_position_entries()
+                try:
```

Comment:
Do we need this? Seems a little hacky... If its necessary, can we have a specific exception to catch, thanks

Reply:
Seriously what is your problem with exceptions based on completely unknown, this is totally fine.

Reply:
Changed it to output the exception in terminal.